applications/byte_patrol/tests.py

```
@@ -1,2 +1,377 @@
+            )
+            logger.debug(
+                f"HTTP_X_FORWARDED_FOR in real request: {request.META.get('HTTP_X_FORWARDED_FOR')}"
+            )
+            return original_process_request(request)
+
+        # Temporarily replace the middleware's process_request method
+        middleware.process_request = debug_process_request
+
+        # Make the request with explicit HTTP headers
+        logger.debug("Using custom test client to make request...")
+        self.client.handler._middleware_chain  # Access to trigger debug logger.debug
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)